test/rest_api/test_compactionhistory.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import requests
+import sys
+from collections import defaultdict
+
+# Use the util.py library from ../cql-pytest:
+sys.path.insert(1, sys.path[0] + '/test/cql-pytest')
+from util import new_test_table, new_test_keyspace
+
+
+def test_compactionhistory_rows_merged_task(cql, rest_api):
+    with new_test_keyspace(cql, "WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1}") as ks:
```

Comment:
Don't use SimpleStrategy in new code, it's deprecated. Use NetworkTopologyStrategy.